README.md

```diff
@@ -2,23 +2,24 @@
 
 ## Coding Guidelines
 
-1. [Project Structure] (ProjectStructure.md)
-2. [Java Guidelines] (JavaGuidelines.md)
-3. [Resources Guidelines] (ResourcesGuidelines.md)
-4. [Android Manifest Guidelines] (Manifest.md)
-5. [Gradle Best Practices] (Gradle.md)
+1. [Project Structure](ProjectStructure.md)
+2. [Java Guidelines](JavaGuidelines.md)
+3. [Resources Guidelines](ResourcesGuidelines.md)
+4. [Android Manifest Guidelines](Manifest.md)
+5. [Gradle Best Practices](Gradle.md)
 
-## Using library
+## Using Libraries
 
-1. [Butterknife] (Libraries/Butterknife.md)
+1. [Butterknife](Libraries/Butterknife.md)
 
-## Being consistent
+## Being Consistent
 
-Our parting thought: BE CONSISTENT. If you're edition code, take a few minutes to look at the code around you and determine its style. If they use spaces around if clauses, you should too. If their comments have little boxes of stars around them, make your comments have little boxes of stars around them too.
+Our parting thought: **BE CONSISTENT**. If you're writing/editing code, take a few minutes to look at the code around you and determine it's style. If the code use spaces around if clauses, you should too. If the comments have little boxes of stars around them, make your comments have little boxes of stars around them too.
 
-The point of having style guidelines is to have a common vocabulary of coding, so people can concentrate on what you're saying, rather than on how you're saying it. We present global style rules here so people know the vocabulary. But local style is also important. If code you add to a file looks drastically different from the existing code around it, it throws readers our of their rhythm when they to to read it.
+The point of having style guidelines is to have a common vocabulary of coding, so people can concentrate on what you're saying, rather than on how you're saying it. We present global style rules here so people know the vocabulary. But local style is also important. If the code you add to a file looks drastically different from the existing code around it, it throws readers our of their rhythm when they to read it.
 
 ## Sources
+
 - https://github.com/futurice/android-best-practices#gradle-configuration (Prefer to follow this)
 - https://lftechnology.atlassian.net/wiki/display/AS/Android+Station
```

Comment:
This page doesn't work. It needs to be accessible by everyone. It's only usable internally.